test/lru-cache.js

```diff
@@ -215,8 +216,52 @@ function makeTests(Cache, name) {
 
       assert.deepStrictEqual(entries, Array.from(cache.entries()));
     });
+
+    if (name === 'ObliviousLRUCache') {
+      it('should be possible to delete keys from a LRU cache.', function() {
```

Comment:
I am wondering whether a rolling test should be done? Like add 2 remove 1, in a loop of ~10 items for a cache of ~4-5 capacity.